bundles/org.openhab.binding.livisismarthome/src/main/java/org/openhab/binding/livisismarthome/internal/util/Translator.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.livisismarthome.internal.util;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link Translator} is responsible for translating strings to another (supported) language.
+ *
+ * @author Sven Strohschein - Initial contribution
+ */
+@NonNullByDefault
+public interface Translator {
```

Comment:
After going through my "@text/" comments, please consider if this class is still needed.

Reply:
solved (class removed)